src/components/todoList/TodoListForm.tsx

```diff
@@ -53,16 +56,16 @@ export function TodoForm() {
           className={darkModeStatus ? 'addtask-dark' : 'addtask-light'}
           type="text"
           name="inputGoal"
-          placeholder="What do you want to achieve today?"
+          placeholder={t('addGoalPlaceholder')}
           value={formInputData.inputGoal}
           onChange={handleChange}
         />
       </div>
       <div>
         <input
           className={darkModeStatus ? 'addtaskdark-time' : 'addtasklight-time'}
-          type="date"
-          placeholder="Add time"
+          type="text"
+          placeholder={t('addTimePlaceholder')}
           name="inputTime"
           value={formInputData.inputTime}
           onChange={handleChange}
```

Comment:
Refer to #213 for the UI snippet

Reply:
I didn't realise that there was a brand new design....I'll close this PR and work on the design tomorrow.